src/openzaak/tests/test_help_text.py

```diff
@@ -0,0 +1,15 @@
+# SPDX-License-Identifier: EUPL-1.2
+# Copyright (C) 2020 Dimpact
+from django.test import TestCase
+
+from openzaak.utils.help_text import mark_experimental
+
+
+class HelptextTests(TestCase):
+    def mark_experimental_test(self):
```

Comment:
Nice to have such a case!

For the future: if you don't need an access to DB in your testcase you can use `uniitest.TestCase` class, which is a little faster than the django `TestCase`